DimaDK24/orders-manager-frontend
# Refactor: eliminate deprecated css files

[Trello ticket](https://trello.com/c/qi6s57nK)

# Problem statement

Files with very generic styles, which applies to the whole app.

# Steps to test the changes

Open main page. Check that everything looks fine both in desktop and mobile versions.

# Solution description

Move styles to appropriate components, remove some styles, refactor some styles.

# Sanity checklist


# Review checklist
